kartothek/io/testing/delete.py

```diff
@@ -20,6 +20,39 @@ def test_delete_dataset(store_factory, metadata_version, bound_delete_dataset):
     assert len(list(store.keys())) == 0
 
 
+def _test_gc(uuid, store_factory, garbage_collect_callable):
+    store = store_factory()
+
+    keys_before = set(store.keys())
+
+    # Add a non-tracked table file
+    store.put("{}/core/trash.parquet".format(uuid), b"trash")
+
+    # Add a non-tracked index file
+    store.put("{}/indices/trash.parquet".format(uuid), b"trash")
+
+    garbage_collect_callable(uuid, store_factory)
```

Comment:
The *delete* tests shoulnd't call the garbage collect but the *delete pipelines* should handle this.

Reply:
removed the garbage collect calls in the delete tests

Reply:
@mganesh1308 did you push your changes, yet?

Reply:
Pushed the changes again..